### PR TITLE
dz4

### PR DESCRIPTION
**Pull request by annse**

### CODE REVIEW

File: src/lesson2/Main.java [hunk 1]
```diff
@@ -1,3 +1,5 @@
+package lesson2;
+
 import java.util.Scanner;
```
Comment (shaitanaleksander) [inline src/lesson2/Main.java:4]: this file is not related to homework 3

**Review by shaitanaleksander — APPROVED** [review]

File: src/lesson2/Main.java (hunk 1 above)
Comment (annse) [reply src/lesson2/Main.java:4]: the project architecture has been changed
